README.md

```diff
@@ -12,10 +12,4 @@ In this application:
   |- LICENCE.md
   |- PULL_REQUEST_TEMPLATE.md
 
-```
-  ## How to run the template site on your local computer
```

Comment:
Not sure if it makes sense to delete this until we replace it with the updated instructions

Reply:
I had it removed because it isn't updated, and someone might follow these instructions here and get confused.